### ISSUE TITLE
Calculate Pelvis initial position from all 4 markers, not just 2. 

### ISSUE THREAD

**Issue by JulienGroud** [AnyBody/anymocap#4]
Some of my C3D files got a kinematic error if the pelvis location is not set this way...

**Comment by melund** [comment]
Thanks a lot for reporting @JulienGroud. I am not sure I understand the issue completely.

The class template to set initial positions uses three markers, not just just two. See for example: 
https://github.com/AnyBody/anymocap/blob/master/Examples/Plug-in-gait-FullBody/TrialSpecificData.any#L39

**Comment by JulienGroud** [comment]
Hi @melund ,
Sorry for the lack of details.

In "AutomaticInitialPositionOfBody.any", I found the line below:

PelvisPos = 0.5*(.Pelvis_RASIS + .Pelvis_LASIS);

However, this results in a pelvis initial position guess that is too much forward and for some of my projects, the kinematic analysis failed.
I changed it to something like 

PelvisPos = 0.25*(.Pelvis_RASIS + .Pelvis_LASIS + .Pelvis_RPSIS + .Pelvis_LPSIS); 

and it worked better so I thought it was worth sharing.
